src/share/poudriere/ports.sh

```diff
@@ -188,10 +188,14 @@ check_portsnap_interactive() {
 
 if [ ${CREATE} -eq 1 ]; then
 	# test if it already exists
-	porttree_exists ${PTNAME} && err 2 "The ports tree, ${PTNAME}, already exists"
-	maybe_run_queued "${saved_argv}"
 	: ${PTMNT="${BASEFS:=/usr/local${ZROOTFS}}/ports/${PTNAME}"}
 	: ${PTFS="${ZPOOL}${ZROOTFS}/ports/${PTNAME}"}
+	porttree_exists ${PTNAME}
+	if [ $? -eq 0 ]; then
```

Comment:
This won't work right since we use `set -e`. A non-0 return from  `porttree_exists` would exit the script immediately (via the traps we have). It needs to use `if porttree_exists ${PTNAME}; then`